cli/cmd/backup/create.go

```diff
@@ -2,33 +2,76 @@ package backup
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/grtl/mysql-operator/cli/config"
+	"github.com/grtl/mysql-operator/cli/options"
+	crv1 "github.com/grtl/mysql-operator/pkg/apis/cr/v1"
 )
 
-var clusterName string
+var (
+	clusterName string
+	backupName  string
+	storage     string
+	time        string
+)
 
 var backupCreateCmd = &cobra.Command{
 	Use:   "create [backup name]",
-	Short: "short description of backup create",
-	Long: `A longer description of backup create with examples:
+	Short: "Schedules a new MySQL backup",
+	Long: `Schedules a new MySQL backup for specified cluster. The backups will
+be created according to cron-style time provided.
 msp backup create "my-backup" --cluster "my-cluster"`,
 	Run: func(cmd *cobra.Command, args []string) {
-		backup := "mysql-backup"
+		options := options.ExtractOptions(cmd)
+
+		backupName = clusterName
 		if len(args) >= 1 {
-			backup = args[0]
+			backupName = args[0]
```

Comment:
We should add a constraint that user can provide only one or zero arguments (`Args:` validator for backupCreateCmd)

Reply:
Done,